packages/material-ui/src/CircularProgress/CircularProgress.js

```diff
@@ -36,6 +36,7 @@ export const styles = theme => ({
   /* Styles applied to the root element if `variant="indeterminate"`. */
   indeterminate: {
     animation: 'mui-progress-circular-rotate 1.4s linear infinite',
+    animationName: '$mui-progress-circular-rotate',
```

Comment:
you can use animation as well

Reply:
Would this be backward compatible with JSS 9.8.7?

Reply:
nope, there was no scoped animation names before and now they are scoped by default, thats why we packed it into v10 and added to breaking changes list in the changelog https://github.com/cssinjs/jss/blob/master/changelog.md#breaking-changes

Reply:
I think the problem is the core package is on the old version of JSS whilst styles is on the alpha

Reply:
Just to confirm that the change is backward compatible between JSS 9 and 10.
I like @kof's suggestion. I'm gonna add a comment so we don't forget about it with the v4 release.

![capture d ecran 2018-12-31 a 16 47 57](https://user-images.githubusercontent.com/3165635/50563295-f1971800-0d1b-11e9-8cd5-179ecc87c283.png)

Reply:
So it's backwards compatible if you use animation-name and animation together? but not if you just use animation?

Reply:
Yes.

Reply:
Thought so :) That’s what I found from my testing